person.rb

```diff
@@ -0,0 +1,30 @@
+# class for person
+class Person
+  attr_accessor :name, :age
+  attr_reader :id
+
+  def initialize(age, name = 'unknown', parent_permission: true)
+    @id = Random.rand(1..1000)
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+
+  def of_age?
+    return true if @age >= 18
+
+    false
+  end
```

Comment:
- [ ] Kindly make the method `of_age?` private as per requirement.

Reply:
Thanm you for your review, I have made the changes required.